functions/routes/ecom/auth-callback.js

```diff
@@ -10,9 +10,22 @@ module.exports = ({ appSdk }) => (req, res) => {
   // https://github.com/ecomclub/ecomplus-app-sdk
   appSdk.handleCallback(storeId, req.body)
     .then(({ isNew, authenticationId }) => {
-      if (isNew) {
-        appSdk.saveProcedures(storeId, procedures)
+      if (!isNew && procedures && procedures.length) {
+        const { triggers } = procedures[0]
+        if (triggers && triggers.length) {
+          return appSdk.getAuth(storeId, authenticationId).then(auth => {
+            const { row, doc } = auth
```

Comment:
```suggestion
            const { row, docRef } = auth
```

https://github.com/ecomclub/ecomplus-app-sdk/commit/0e4e8b6c74dcfaa09898b329ee87b1aaf08233dd